knn-transformers.ipynb

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
<p>Maybe something like "explicitly setting the <code>transformer</code> parameter" allows customization and bypasses Scanpy's shortcut for small datasets. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scanpy-tutorials/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>